src/Rules/Methods/CallMethodsRule.php

```diff
@@ -31,12 +37,34 @@ public function getNodeType(): string
 
 	public function processNode(Node $node, Scope $scope): array
 	{
-		if (!$node->name instanceof Node\Identifier) {
-			return [];
+		$errors = [];
+		if ($node->name instanceof Node\Identifier) {
+			$methodNameScopes = [$node->name->name => $scope];
+		} else {
+			$nameType = $scope->getType($node->name);
+			$methodNames = array_map(static fn (ConstantStringType $type): string => $type->getValue(), $nameType->getConstantStrings());
+			$methodNameScopes = array_column(array_map(
```

Comment:
Please write less clever code without multiple array_map and array_column calls. A simple foreach would suffice!